### ISSUE TITLE
Does this app even work on Windows 11 at all?

### ISSUE THREAD

**Issue by MrManinBlack1337** [gabe565/CastSponsorSkip#58]
![cc](https://github.com/gabe565/CastSponsorSkip/assets/87234960/5b3aff73-0cbd-4f22-99c9-70d55379009b)

This is the only thing I get after running the exe. Nothing happens after that even after starting my CC, playing youtube videos or even starting CC first instead of CastSponsorSkip. The help document mentions nothing of Windows version.

Oh I am running chromecast v2 if this helps. What am I missing here? Thanks

**Comment by gabe565** [comment]
It should work with Windows, but I haven't tested it myself yet. If it's not too much trouble, can you install [go-chromecast](https://github.com/vishen/go-chromecast) and run `go-chromecast ls` to see if any devices are detected?

**Comment by MrManinBlack1337** [comment]
![docker castsponsorskip](https://github.com/gabe565/CastSponsorSkip/assets/87234960/9a65d35b-88c5-46ac-8e11-78a88a525b99)

So I went ahead and install docker but not for this test, it was for ISponsorBlockTV. Whille it works, yours does not, It still stuck on searching for devices. I really wish this simple exe one works because it does not make any sense to use docker which consumes GIGS of SSD space and RAM just to run this thing. Unfortunately ISponsorBlockTV does not have any simple exe builds.

oh and ignore the "exiting" I was installing yours from command prompt and rerunning it on docker window to capture screenshot

**Comment by MrManinBlack1337** [comment]
![go chromecast](https://github.com/gabe565/CastSponsorSkip/assets/87234960/0e0d4745-d4b2-4d80-bd0d-7ca0fdb49540)

I saw the go chromecast thingie and its just an exe so I ran it. This is running while my TV/CC is on

**Comment by gabe565** [comment]
@MrManinBlack1337 Thanks for testing that. It seems like discovery may not work from Windows. I'll have to do some additional testing to see if I can get it working.

In the mean time, I've added support for hardcoding the device list in v0.7.0! A device list can be passed with the new `--devices` flag or with the `CSS_DEVICES` env.

**Comment by MrManinBlack1337** [comment]
This works! I can live with this. Way better than running resources heavy docker. Anyway can I connect to multiple devices or just one while using this flag?

**Comment by gabe565** [comment]
@MrManinBlack1337 Yes! I should have specified in my original comment, but that flag/env accepts a comma-separated list of device IP addresses
